AirCasting/SearchAndFollow/SearchAndFollowMap/SearchAndFollowMap.swift

```diff
@@ -118,7 +132,34 @@ struct SearchAndFollowMap: UIViewRepresentable {
                 
             }
             parent.startingPoint = CLLocationCoordinate2D(latitude: mapView.camera.target.latitude, longitude: mapView.camera.target.longitude)
-            dotsPositionShouldChange(for: mapView)
+            startingPointHolder = parent.startingPoint
+            if parent.startingPointChanged {
+                startPositionShouldChange(for: mapView)
+                DispatchQueue.main.async { self.parent.startingPointChanged = false }
+            } else {
+                dotsPositionShouldChange(for: mapView)
+            }
```

Comment:
the `else` is still not needed if we put a `return` at the end of the `if` block